### PR TITLE
Default layout

### PR DESCRIPTION
**Pull request by Vladimir-csp**
Add an optional argument for default layout index.

Apply `black` formatter.

### CODE REVIEW

File: layout-per-window.py [hunk 1]
```diff
@@ -1,49 +1,76 @@
 #!/usr/bin/env python
 
 # This script keeps track of the active layout for each window.
+# Optional argument defines numeric layout index for new windows (counted from 0)
 #
 # This script requires i3ipc-python package (install it from a system package
 # manager or pip).
 
+import sys
```
Comment (OctopusET) [inline layout-per-window.py:9]: Need a fix

Comment (Vladimir-csp) [reply layout-per-window.py:9]: Does this mean putting sys import after i3ipc-python? I thought third-party imports should come after stdlib imports.

Comment (OctopusET) [reply layout-per-window.py:9]: Just a newline will do, it's a lint error

File: layout-per-window.py [hunk 2]
```diff
@@ -1,49 +1,77 @@
...
+        if sys.argv[1].isnumeric():
+            default_layout = int(sys.argv[1])
+        else:
+            print(f"Expected an integer, got: {sys.argv[1]}", file=sys.stderr)
+            sys.exit(2)
+    elif len(sys.argv) > 2:
+        print("Too many arguments", file=sys.stderr)
+        sys.exit(2)
+    else:
+        default_layout = None
```
Comment (Vladimir-csp) [inline layout-per-window.py]: What annotation would suffice here? `int` or `typing.Optional[int]`?

Comment (OctopusET) [reply layout-per-window.py]: let's just use `int`

**Comment by OctopusET** [comment]
I think it would be better to test your codes on local. Try install mypy

**Review by OctopusET — CHANGES_REQUESTED** [review]
Thank you! Would you squash commits into one.

**Comment by Vladimir-csp** [comment]
> Would you squash commits into one.

Done.

**Review by OctopusET — APPROVED** [review]
Thanks LGTM

**Pull request merged**